pygmt/src/hlines.py

```diff
@@ -0,0 +1,164 @@
+"""
+hlines - Plot horizontal lines.
+"""
+import numpy as np
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    build_arg_string,
+    data_kind,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    D="offset",
+    J="projection",
+    N="no_clip",
+    R="region",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="zvalue",
+    l="label",
+    p="perspective",
+    t="transparency",
```

Comment:
IMHO, we should keep the function simple, and do not add aliases like `frame`, `projection`, `region`, `timestamp`, `xshift`, `yshift`, `zvalue`, `cmap`.

For people who want coloring lines based on z values (`zvalue` and `cmap`), they should use `Figure.plot()` for more features.

Reply:
Ok, would be fine for me @seisman ! Means it should be only possible to plot lines with colors defined in `pen` and all pepole who want to plot advanced and fancier stuff should take a look at `Figure.plot()`?

Reply:
Yes.